src/modules/plus/filter_dynamicgpstext.yml

```diff
@@ -0,0 +1,321 @@
+schema_version: 0.1
+type: filter
+identifier: dynamicgpstext
```

Comment:
I wonder about the name "dynamicgpstext". I think it could just be "gpstext". This would apply to the identifier as well as the file names and filter naming.

Reply:
I'm ok with gpstext. I was using "GPS text overlay" as a filter name in qml but now that I think of it, the overlay word applies to all filters so it's useless.

Reply:
I agree. sometimes less is more.